lib/content-services/src/lib/content-metadata/components/content-metadata/content-metadata.component.html

```diff
@@ -5,7 +5,7 @@
                              [expanded]="canExpandProperties()"
                              [attr.data-automation-id]="'adf-metadata-group-properties'">
             <mat-expansion-panel-header>
-                <mat-panel-title role="heading">
+                <mat-panel-title role="heading" aria-level="3">
```

Comment:
I see here still some places which are using aria-level, can you use h1, h2 etc. everywhere instead of aria-level in that PR? I think it should be possible if you do it like: 
<mat-panel-title>
  <h3>{{ 'CORE.METADATA.BASIC.HEADER' | translate }}</h3>
</mat-panel-title>

Reply:
I resolved with heading tags instead of div

Reply:
Thank you. @jatin2008 I think you can also remove now these role"heading" right? No need to say that it is heading if you use h1/h2 etc. which already mean "heading"

Reply:
noted and resolved